### PR TITLE
v5.0.2

### PR DESCRIPTION
**Pull request by rekmarks**
Patch version bump before we merge in a bunch of breaking changes for v6 (#42).

### CODE REVIEW

**Review by whymarrh — DISMISSED** [review]

**Review by Gudahtt — CHANGES_REQUESTED** [review]
The changelog should be updated as well

**Comment by Gudahtt** [comment]
Oh, the changelog update was missed with v5.0.1 as well it seems

**Comment by rekmarks** [comment]
@Gudahtt changelog updated!

**Review by Gudahtt — DISMISSED** [review]
LGTM!

**Comment by rekmarks** [comment]
@Gudahtt done

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,29 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [5.0.2] - 2020-05-22
+
+### Changed
+
+- Fix `ethereum.send` return value for certain argument combination
```
Comment (Gudahtt) [inline CHANGELOG.md:14]: Nit: It'd be useful to link the relevant for each of these entries as well

**Review by Gudahtt — APPROVED** [review]
LGTM!

File: CHANGELOG.md (hunk 1 above)
Comment (rekmarks) [reply CHANGELOG.md:14]: I'll do that in a follow-up!

**Pull request merged**